core/orch_test.go

```diff
@@ -527,6 +530,7 @@ func TestProcessPayment_GivenNilNode_ReturnsNilError(t *testing.T) {
 
 func TestProcessPayment_GivenNilRecipient_ReturnsNilError(t *testing.T) {
 	n, _ := NewLivepeerNode(nil, "", nil)
+	n.Balances = NewBalances(5 * time.Second)
```

Comment:
Do we to initialize this here? `ProcessPayment()` won't access this before returning here

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/956/commits/794632acb80531d8222ca2a12334d3ddcd321427